packages/langium/src/lsp/call-hierarchy-provider.ts

```diff
@@ -54,12 +54,12 @@ export abstract class AbstractCallHierarchyProvider implements CallHierarchyProv
             return undefined;
         }
 
-        const declarationNode = this.references.findDeclarationNode(targetNode);
+        const declarationNode = this.references.findDeclaration(targetNode);
```

Comment:
will this introduce a breaking change of public API?

Reply:
We don't follow semantic versioning, since every change in an internal class (which we need to expose due to the override mechanism) would require a new major version.